requirements/base.txt

```diff
@@ -15,8 +15,6 @@ apispec[yaml]==6.3.0
     # via flask-appbuilder
 apsw==3.42.0.1
     # via shillelagh
-async-timeout==4.0.3
```

Comment:
just noting that this particular pinned dep has been flickering, meaning `pip-compile-multi` isn't as deterministic as I'd like for it to be, for reasons I don't really understand